include/tudat/interface/spice/spiceInterface.h

```diff
@@ -136,6 +136,18 @@ std::vector<std::string> getStandardSpiceKernels(const std::vector<std::string>
 void loadStandardSpiceKernels(const std::vector<std::string> alternativeEphemerisKernels =
                                   std::vector<std::string>());
 
+void toggleErrorReturn( );
+
+void toggleErrorAbort( );
+
+void suppressErrorOutput( );
+
+std::string getErrorMessage( );
+
+bool checkFailure( );
+
```

Comment:
We should more properly implement the spice error handling in Tudat, but for now, this will do. Let's open an issue, and discuss how to best do this